core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_checkbox.scss

```diff
@@ -1,107 +1,62 @@
 @use "variables" as *;
 
-p-checkbox.ng-dirty.ng-invalid > .p-checkbox > .p-checkbox-box {
-    border-color: $error;
-}
-
-@keyframes checkbox-check {
-    0% {
-        width: 0;
-        height: 0;
-        border-color: $white;
-        transform: translate3d(0, 0, 0) rotate(45deg);
-    }
+p-checkbox.p-element {
+    gap: $spacing-1;
 
-    33% {
-        width: 4px;
-        height: 0;
-        transform: translate3d(0, 0, 0) rotate(45deg);
+    &.ng-dirty.ng-invalid {
+        .p-checkbox {
+            border-color: $color-alert-red;
+        }
     }
 
-    100% {
-        width: 4px;
-        height: 10px;
-        border-color: $white;
-        transform: translate3d(0, -10px, 0) rotate(45deg);
+    .p-checkbox {
+        height: 24px;
+        aspect-ratio: 1/1;
+        border-radius: $border-radius-sm;
+        border: 2px solid $color-palette-gray-400;
+
+        &.p-checkbox-checked {
+            border-color: $color-palette-primary-500;
+        }
+
+        &.p-checkbox-disabled {
+            border-color: $color-palette-gray-400;
+            background-color: $color-palette-gray-200;
+
+            .p-checkbox-box.p-highlight {
+                .p-checkbox-icon {
+                    color: $color-palette-gray-400;
+                }
+            }
+        }
+
+        .p-checkbox-box {
+            height: 20px;
+            aspect-ratio: 1/1;
```

Comment:
Are you sure we need this?

Reply:
Yes, otherwise the box won't grow, I set it to 20px to leave some space between the border and the actual box